sentence_transformers/sparse_encoder/SparseEncoder.py

```diff
@@ -215,6 +217,8 @@ def encode(
             save_to_cpu (bool, optional):  Whether the output should be moved to cpu or stay on the device it has been computed on.
                 Defaults to False
             device (str, optional): Which :class:`torch.device` to use for the computation, if None current device will be use. Defaults to None.
+            max_active_dims (int, optional): The maximum number of active dimensions in the output of the model. `None` means we will used the value of the
+                model's config. Defaults to None.
 
         Returns:
             Union[List[Tensor], ndarray, Tensor]: By default, a 2d numpy array with shape [num_inputs, output_dimension] is returned.
```

Comment:
Let's remove the `If ``self.truncate_dim <= output_dimension`` then output_dimension is ``self.truncate_dim``.`, you can also explain the ``convert_to_sparse_tensor`` here if you think it fits